plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -0,0 +1,611 @@
+/*
+ * Copyright (C) 2020 Jimmy Yu <Jimmy_yu@pixart.com>
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#ifdef HAVE_HIDRAW_H
+#include <linux/hidraw.h>
+#include <linux/input.h>
+#endif
+
+#include "fu-common.h"
+#include "fu-chunk.h"
+
+#include "fu-pxi-device.h"
+#include "fu-pxi-firmware.h"
+
+#define PXI_HID_DEV_OTA_INPUT_REPORT_ID		0x05
+#define PXI_HID_DEV_OTA_OUTPUT_REPORT_ID	0x06
+#define PXI_HID_DEV_OTA_FEATURE_REPORT_ID	0x07
+
+#define FU_PXI_DEVICE_CMD_FW_OTA_INIT		0x10u
+#define FU_PXI_DEVICE_CMD_FW_WRITE		0x17u
+#define FU_PXI_DEVICE_CMD_FW_UPGRADE		0x18u
+#define FU_PXI_DEVICE_CMD_FW_MCU_RESET		0x22u	/* remove after windows OTA app ready */
```

Comment:
Can this comment please be expanded?  Does this mean that the reset should go away when a future firmware comes that can handle the upgrade differently?  Leaving a comment like this makes me worry about maintainability.

Reply:
@jyu99 can you explain please

Reply:
It’s for early version using ,so I will remove it. Thank you suggestions

Reply:
you mean remove the _comment_? right?

Reply:
Yes.. remove the comment